prdoc/prdoc/pr_6715.prdoc

```diff
@@ -0,0 +1,14 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Update Nomination Pool Pallet to Support Block Number Provider
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      This PR makes the nomination pool pallet use the relay chain as a block provider for a parachain on a regular schedule.
+      To migrate existing nomination pool implementations, simply add `type BlockNumberProvider = System` to have the same behavior as before.
+
+crates:
+- name: pallet-nomination-pools
+  bump: minor
```

Comment:
what is the official versioning used here?
If we follow rust semver, then it is a breaking change, so major.
But if we consider how much it is changing the API then I agree it is minor.

Reply:
i have done as suggested